src/ImageAPI/LambdaFunction.cs

```diff
@@ -39,17 +40,36 @@ public APIGatewayProxyResponse Run(APIGatewayProxyRequest input)
         var databaseItem = dynamoDb.GetItemAsync(hash)
             .GetAwaiter()
             .GetResult();
+
         if (databaseItem.Item.Count == 0)
         {
             return ResponseBuilder.WrongHash();
         }
 
+        if (databaseItem.Item.TryGetValue("Image", out var base64Image))
+        {
+            return new APIGatewayProxyResponse
+            {
+                IsBase64Encoded = true,
+                StatusCode = (int)HttpStatusCode.OK,
+                Body = base64Image.S,
+                Headers = new Dictionary<string, string>
+                {
+                    { "Content-Type",  databaseItem.Item["Content-Type"].S}
+                }
+            };
+        }
+
         var attributes = JsonConvert.DeserializeObject<Erc721Attribute[]>(databaseItem.Item["Data"].S)!;
 
         try
         {
             var providerImage = ProviderImageFactory.Create(backgroundImage, font, attributes);
 
+            dynamoDb.UpdateItemAsync(hash, providerImage.Base64Image, databaseItem.Item["Content-Type"].S)
```

Comment:
Need to use `providerImage.ContentType` instead of `databaseItem.Item["Content-Type"].S`